index.js

```diff
@@ -0,0 +1,47 @@
+var fizzbuzz = (function() {
+  var _fizzBuzz = function(fizz, buzz) {
+    this.fizzbuzz = fizz + buzz || "fizzbuzz";
+    this.fizz = fizz || "fizz";
+    this.buzz = buzz || "buzz";
+    this.nums = [];
+  };
+
+  _fizzBuzz.prototype = {
+    input: function(low, high) {
+
+      this.low = low;
+      this.high = high;
+
+      this.nums.push(low);
+      this.nums.push(high);
+
+    return this.nums;
```

Comment:
This is a nice touch, in case a developer working with your code wants to be able to capture the input values for re-use later on.